### ISSUE TITLE
Running JMC, Eclipse with jenv and OpenJDK 14 results in missing jimage library error

### ISSUE THREAD

**Issue by deors** [jenv/jenv#302]
Hello,

When running JMC or Eclipse IDE (likely other Eclipse-based apps, too), from command line on top of jenv, and OpenJDK 14 selected (both java.net and AdoptOpenJDK distros), results in the following error:

```
Error occurred during initialization of VM
Unable to load jimage library: /Library/Java/JavaVirtualMachines/adoptopenjdk-14.jdk/Contents/Home/bundle/libjimage.dylib
```

After thorough testing and the support of AdoptOpenJDK community (see [here](https://github.com/AdoptOpenJDK/openjdk-support/issues/123) for reference), it seems that the root cause lies with jenv custom script in shims folder or with jenv exec itself.

Scenario 1 - error:
Path: /Users/xxx/.jenv/shims:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
Java_home: /Users/jorgehidalgo/.jenv/versions/14

Scenario 2 (no shims) - works fine:
Path: /Users/jorgehidalgo/.jenv/versions/14/bin:/usr/bin:/bin:/usr/sbin:/sbin
Java_home: /Users/jorgehidalgo/.jenv/versions/14

Scenario 3 (no jenv in use) - also works:
Path: /Library/Java/JavaVirtualMachines/adoptopenjdk-14.jdk/Contents/Home/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
Java_home: /Library/Java/JavaVirtualMachines/adoptopenjdk-14.jdk/Contents/Home

So, in a nutshell, I think that the root cause may be located in shims script or in jenv executable itself.
Other uses of jenv, like running Java programs from the shell, or through Maven/Gradle, work as expected.

I'm open to help with debugging, if that's useful.

BR, J.

**Comment by OlofSandell** [comment]
I can confirm this bug, and find it quite urgent to fix, at least until someone present a viable workaround.
